Apps/AdminWebClient/src/ClientApp/src/utility/phnValidator.ts

```diff
@@ -2,8 +2,7 @@ const PHNsigDigits: number[] = [2, 4, 8, 5, 10, 9, 7, 3];
 
 export default abstract class PHNValidator {
     public static IsValid(PHN: string): boolean {
-        const isNumeric: boolean = /^\d+$/.test(PHN);
-        if (PHN.length != 10 || !isNumeric) {
+        if (PHN.length != 10 || isNaN(PHN)) {
```

Comment:
isNaN only compares against the JavaScript NaN value, which would only be set if we were performing bad arithmetic operations with the value.  It won't tell us whether or not we are dealing with a string of digits.